control-plane-operator/controllers/hostedcontrolplane/cloud/powervs/powervs.go

```diff
@@ -155,7 +155,7 @@ func ReconcileCCMDeployment(deployment *appsv1.Deployment, hcp *hyperv1.HostedCo
 						},
 						VolumeMounts: []corev1.VolumeMount{
 							{
-								Name:      hcp.Status.KubeConfig.Name,
+								Name:      "service-network-admin-kubeconfig",
```

Comment:
nit: pls use manifests.KASServiceKubeconfigSecret("").Name
That makes it easier to identify the components that reference that kubeconfig

Reply:
thanks for pointing out, fixed. PTAL